acp/acp.go

```diff
@@ -43,6 +43,9 @@ type ACP interface {
 	// Close closes the resources in use by acp.
 	Close() error
 
+	// DropAll purges the entire ACP state.
+	DropAll(ctx context.Context) error
```

Comment:
suggestion: As mentioned on discord :P prefer `ResetState` over `DropAll`

```suggestion
	// ResetState resets the entire ACP state.
	ResetState(ctx context.Context) error
```

Reply:
Would `Reset` work, or do you prefer the full `ResetState`?

Reply:
Reset is fine too